layout/example_test.go

```diff
@@ -127,6 +127,30 @@ func ExampleList() {
 	// 5
 }
 
+func ExampleContain() {
+	gtx := layout.Context{
+		Ops: new(op.Ops),
+		// Rigid constraints with both minimum and maximum set.
```

Comment:
These constraints don't seem rigid to me.

Reply:
Fixed.